src/pages/Organization/AdminLogin.tsx

```diff
@@ -49,9 +49,10 @@ function AdminLogin() {
           await client.resetStore();
           toast.success(t(`Welcome`));
           if (state) {
-            navigate(`${state}`);
-          } else {
-            navigate('/dashboard/');
+            //navigate to ${state},in case you want to make it default (/dashboard),  
+            {data.loginUser.user.role === 'manager' || 'user'? navigate(`/dashboard`): navigate('/dashboard/trainees')}
```

Comment:
Why are we redirecting `manager`  and `user` role to dashboard?

Reply:
Because there are no trainees on their dashboard, i chose to remain on the dashboard. do you have any suggestions where we can redirect them